### PR TITLE
Sort table view columns alphabetically

### LINKED ISSUE #2638: Columns are no longer alphabetically ordered in the table view
Reporting as you just suggested me in the office. :smile:

### PR DESCRIPTION
**Pull request by viddo**
Fixes #2638

- Changed the `sortSchema` method to sort "default" items alphabetically i.e. items that should not be on a specific position according to the predefined priorities
- Removed the optional priorities param since it was not used anywhere

screen shot 2015-03-18 at 15 48 19

### CODE REVIEW

**Comment by viddo** [comment]
@javisantana ok?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2435/))

File: lib/assets/javascripts/cartodb/models/table.js [hunk 1]
```diff
@@ -178,8 +178,8 @@
       return name && name.replace(/"/g, '');
     },
 
-    sortSchema: function(priorities) {
-      this.set('schema', cdb.admin.CartoDBTableMetadata.sortSchema(this.get('schema'), priorities));
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb/models/table.js]: why did you remove the param?

Comment (viddo) [inline lib/assets/javascripts/cartodb/models/table.js]: > - Removed the optional priorities var since it was not used anywhere

Comment (javisantana) [inline lib/assets/javascripts/cartodb/models/table.js]: I mean, I'm probably agree, but not sure why I did that, probably we were using that param at some point from somewhere

**Comment by javisantana** [comment]
:es:

File: lib/assets/javascripts/cartodb/models/table.js (hunk 1 above)
Comment (viddo) [inline lib/assets/javascripts/cartodb/models/table.js]: Yeah, I figured, in general I think it's better to remove unused code when doing changes though than leaving it around "just in case" :)

Comment (javisantana) [inline lib/assets/javascripts/cartodb/models/table.js]: yes, agree

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2436/))
